### ISSUE TITLE
Trailing comma tuple check misses multi-line tuples

### ISSUE THREAD

**Issue by nivekuil** [PyCQA/pylint#2007]
### Steps to reproduce

```
"""test.py"""
A = ['f',
    ],
print(A)
```
> pylint test.py                                                                          
No config file found, using default configuration
--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

> python test.py                                                                          
(['f'],)

### Current behavior
No error

### Expected behavior
Pylint should complain

### pylint --version output
No config file found, using default configuration
pylint 1.8.3,
astroid 1.6.2
Python 3.6.4 (default, Jan  5 2018, 02:35:40)
[GCC 7.2.1 20171224]

**Comment by dhensen** [comment]
This bit us twice this week, any indication on a time-window for a fix?

**Comment by PCManticore** [comment]
@dhensen No, there's no time window for this fix. If this is important, feel free to take a stab at implementing a PR, but it's pretty low on my priority list.

**Comment by kkew3** [comment]
Another failed case for multiline function call ... got bit today ...

```python
def f(_):
    return None

decider = f(
    0),
```

It would be great if it could be fixed some day. Thanks
